recOrder/scripts/calibrate.py

```diff
@@ -0,0 +1,30 @@
+# This script can be modified to debug and test calibrations
+
+import napari
+import time
+import matplotlib.pyplot as plt
+from recOrder.plugin.widget.main_widget import MainWidget
+from recOrder.calib.Calibration import QLIPP_Calibration
+from recOrder.plugin.workers.calibration_workers import CalibrationWorker
+
+save_dir = "Q:\\Talon\\2022_09_30_calibration_5state\\"
```

Comment:
If we do decide to release this script, maybe clean this?

Reply:
You bet! Cleaned.